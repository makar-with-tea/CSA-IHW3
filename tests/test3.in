N
data/test3.in
0
10
data/test3.out