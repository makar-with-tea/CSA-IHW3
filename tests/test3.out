Show results in the console? Y/N 
Input path to file for reading:
Input first index (n1). It should satisfy the inequality: 0 <= n1 < length of file info 
Input second index (n2). It should satisfy the inequality: n1 <= n2 < length of file info 
Input path to file for writing: 